townhall/myapi/migrations/0020_project_alter_volunteer_password_delete_message.py

```diff
@@ -23,7 +23,7 @@ class Migration(migrations.Migration):
         migrations.AlterField(
             model_name='volunteer',
             name='password',
```

Comment:
I dont know why theres changes here, there shouldnt be. The way to fix this is to go to main locally (git checkout main), update your local main (git pull), then go back to this branch (git checkout DAO-and-service-method-to-get-project-by-ID) then merge main (git merge main) 

Then after do git push to ensure you have the latest changes published

Reply:
Thanks, done now